README.rst

```diff
@@ -495,6 +499,13 @@ Choose from the list of available fixers:
                         An empty line feed should
                         precede a return statement.
 
+* **scalar_cast** [symfony]
```

Comment:
So whay you would say that `bool` is correct and `boolean` is... incorrect?
Both works, one may use only short form, one only long form.